### PR TITLE
neomutt: update to 20211029

### PR DESCRIPTION
**Pull request by x1ddos**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by x1ddos** [comment]
> Have the results of the proposed changes been tested?

```
./xbps-src -Q pkg neomutt
=> neomutt-20211029_1: running do_check ...
test/neomutt-test
...
SUCCESS: All unit tests have passed.
```

**Comment by paper42** [comment]
> > Have the results of the proposed changes been tested?
> 
> ```
> ./xbps-src -Q pkg neomutt
> => neomutt-20211029_1: running do_check ...
> test/neomutt-test
> ...
> SUCCESS: All unit tests have passed.
> ```

This question is meant to be answered by the next two lines with checkboxes, running automated checks is not enough for testing an update.

**Comment by x1ddos** [comment]
sure. updated.

File: srcpkgs/neomutt/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'neomutt'
 pkgname=neomutt
-version=20210205
-revision=2
+version=20211029
+revision=1
 wrksrc="neomutt-${version}"
```
Comment (paper42) [inline srcpkgs/neomutt/template:5]: we can remove this line

**Review by paper42 — COMMENTED** [review]
could you change the commit message to "neomutt: update to 20211029"? If you want to include that the test files were also updated, you can mention it in the commit decription.

File: srcpkgs/neomutt/template (hunk 1 above)
Comment (x1ddos) [reply srcpkgs/neomutt/template:5]: that doesn't seem to work well. the pkg doesn't build with that one and `build_wrksrc="$wrksrc"` a couple lines later removed:

```
/void-packages/common/build-style/configure.sh: line 8: ./configure: No such file or directory
=> ERROR: neomutt-20211029_1: do_configure: '${configure_script} ${configure_args}' exited with 127
=> ERROR:   in do_configure() at common/build-style/configure.sh:8
```

**Comment by x1ddos** [comment]
> could you change the commit message to "neomutt: update to 20211029"?

done. ptal.

File: srcpkgs/neomutt/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/neomutt/template:5]: nevermind, I forgot that this package has multiple distfiles

Comment (paper42) [reply srcpkgs/neomutt/template:5]: We should still be able to remove this line if we replace `build_wrksrc="$wrksrc"` with `build_wrksrc="$pkgname-$version"`

**Comment by x1ddos** [comment]
> We should still be able to remove this line if we replace `build_wrksrc="$wrksrc"` with `build_wrksrc="$pkgname-$version"`

done. ptal.

**Pull request merged**